client/viame-web-common/components/TrackDetailsPanel.vue

```diff
@@ -0,0 +1,394 @@
+            trackSelectNext(1);
+          },
+          disabled,
+        },
+        {
+          bind: 'del',
+          handler: () => {
+            if (selectedTrackIdRef.value !== null) {
+              removeTrack([selectedTrackIdRef.value]);
+            }
+          },
+          disabled,
+        },
+        {
+          bind: 'x',
+          handler: () => trackSplit(selectedTrackIdRef.value, frameRef.value),
+          disabled,
+        },
+      ];
+    });
+
+    onBeforeMount(async () => {
+      attributes.value = await getAttributes();
+    });
+
+    return {
+      selectedTrackIdRef,
+      /* Attributes */
+      attributes,
+      activeSettings,
+      /* Editing */
+      editingAttribute,
+      saveAttribtueHandler,
+      deleteAttributeHandler,
+      editingError,
+      editIndividual,
+      /* Selected */
+      selectedTrack,
+      /* Update functions */
+      closeEditor,
+      editAttribute,
+      addAttribute,
+      confidencePairsAdd,
+      editingModeRef,
+      typeStylingRef,
+      allTypesRef,
+      setEditIndividual,
+      resetEditIndividual,
+      mouseTrap,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-card
+    ref="card"
+    v-mousetrap="mouseTrap"
+    :width="width"
+    class="d-flex flex-column overflow-hidden"
+    @click.native="resetEditIndividual"
+  >
+    <v-subheader>Track Editor</v-subheader>
+    <div
+      v-if="!selectedTrack"
+      class="ml-4 body-2"
+    >
+      No track selected
+    </div>
+    <template v-else>
+      <datalist id="allTypesOptions">
+        <option
+          v-for="type in allTypesRef"
+          :key="type"
+          :value="type"
+        >
+          {{ type }}
+        </option>
+      </datalist>
+      <track-item
+        :single-display="true"
+        :track="selectedTrack"
+        :track-type="selectedTrack.confidencePairs[0][0]"
+        :selected="true"
+        :editing="!!editingModeRef"
+        :input-value="true"
+        :color="typeStylingRef.color(selectedTrack.confidencePairs[0][0])"
+        :lock-types="lockTypes"
+        @seek="$emit('track-seek', $event)"
+      />
+
+      <!-- CONFIDENCE PAIRS -->
+      <div class="border-highlight">
+        <v-row
+          no-gutters
+          class="align-center"
+        >
+          <b>Confidence Pairs:</b>
+          <v-spacer />
+          <v-tooltip
+            open-delay="200"
+            bottom
+            max-width="200"
+          >
+            <template #activator="{ on }">
+              <v-btn
+                disabled
+                outlined
+                x-small
+                class="my-1"
+                v-on="on"
+                @click="confidencePairsAdd"
+              >
+                <v-icon small>
+                  mdi-plus
+                </v-icon>
+                Pair
+              </v-btn>
+            </template>
+            <span>Add a new Confidence Pair</span>
+          </v-tooltip>
+        </v-row>
+      </div>
```

Comment:
Refactor to component maybe.

Reply:
I figured that was going to be the eventual plan, didn't know if it should be done here or in a new PR.  I can do it here.